### ISSUE TITLE
[RFC] InstanceNorm default affine value

### ISSUE THREAD

**Issue by SsnL** [pytorch/pytorch#22755]
TL;DR: the default `affine` value is different from the original paper and code, but is probably too old to change. Can we add a note in doc?

I happened to be looking through the original paper on [Instance Normalization](https://arxiv.org/pdf/1701.02096.pdf) and the accompanied [code](https://github.com/DmitryUlyanov/texture_nets/blob/aad2cc6f8a998fedc77b64bdcfe1e2884aa0fb3e/InstanceNormalization.lua#L16-L20). Both seem to indicate that the affine transform should be on **by default**. E.g., quote from the paper:

> Another similarity with BN is that each IN layer is followed by a scaling and bias operator sx+b.

However, since the very beginning of `nn.InstanceNorm*d`'s existence ([this PR](https://github.com/pytorch/pytorch/pull/1283)), `affine=False` is the default.

Is it worthwhile to add a `.. note::` to the doc about this behavior?

Also, cc @DmitryUlyanov , who is the first author of the paper, owner of the original lua code, and the contributor of the PyTorch module.

**Comment by fmassa** [comment]
In the original PyTorch PR, @DmitryUlyanov (the original author of Instance Norm as well) explicitly mentions
>  allows affine transform, but it is switched off by default.

I suppose there was a reason why it was turned off by default?

**Comment by stalhabukhari** [comment]
@fmassa @DmitryUlyanov I am interested in knowing if using `affine=False` is beneficial in any known case. If not, then I believe the default should be `affine=True`.
